defaults/main.yml

```diff
@@ -227,113 +341,117 @@ roundcube__plugins: []
+# .. envvar:: roundcube__nginx__dependent_servers [[[
 #
 # :program:`nginx` server configuration managed by the debops.nginx_ role.
-roundcube__nginx_server:
-  by_role: 'ansible-roundcube'
-  enabled: True
-  type: 'php5'
-  name: '{{ roundcube__domain }}'
-  root: '{{ roundcube__git_checkout }}'
-  access_policy: '{{ roundcube__nginx_access_policy }}'
-  index: 'index.php'
-
-  options: |
-    autoindex off;
-    client_max_body_size {{ roundcube__max_file_size }}M;
-    client_body_buffer_size 128k;
-
-  location:
-    '/': |
-      try_files $uri $uri/ @roundcube;
-
-    '@roundcube': |
-      rewrite ^/favicon\.ico$       skins/larry/images/favicon.ico    last;
-
-    '~ ^/?(installer|[A-Z0-9]+$)': |
-      deny all;
-
-    '~ ^/?(\.git|\.tx|SQL|bin|config|logs|temp|tests|program\/(include|lib|localization|steps))': |
-      deny all;
-
-    '~ /(README\.md|composer\.json-dist|composer\.json|package\.xml|Dockerfile)$': |
-      deny all;
-
-  php5: 'php5_roundcube'
-  php5_options: |
-    fastcgi_intercept_errors        on;
-    fastcgi_ignore_client_abort     off;
-    fastcgi_connect_timeout         60;
-    fastcgi_send_timeout            180;
-    fastcgi_read_timeout            180;
-    fastcgi_buffer_size             128k;
-    fastcgi_buffers               4 256k;
-    fastcgi_busy_buffers_size       256k;
-    fastcgi_temp_file_write_size    256k;
-
-
+roundcube__nginx__dependent_servers:
+
+  - name: '{{ roundcube__domain }}'
+    filename: 'debops-contrib.roundcube'
+    by_role: 'debops-contrib.roundcube'
+    type: 'php'
+    default: False
+    root: '{{ roundcube__git_checkout }}'
+    access_policy: '{{ roundcube__nginx_access_policy }}'
+    index: 'index.php'
+
+    options: |
+      autoindex off;
+      client_max_body_size {{ roundcube__max_file_size }}M;
+      client_body_buffer_size 128k;
+
+    location_list:
+      - pattern: '/'
+        options: |
+          try_files $uri $uri/ @roundcube;
+
+      - pattern: '@roundcube'
+        options: |
+          rewrite ^/favicon\.ico$ skins/larry/images/favicon.ico last;
+
+      - pattern: '~ ^/?(installer|[A-Z0-9]+$)'
+        options: |
+          deny all;
+
+      - pattern: '~ ^/?(\.git|\.tx|SQL|bin|config|logs|temp|tests|program\/(include|lib|localization|steps))'
+        options: |
+          deny all;
+
+      - pattern: '~ /(README\.md|composer\.json-dist|composer\.json|package\.xml|Dockerfile)$'
+        options: |
+          deny all;
+
+    php_upstream: 'php_roundcube'
+    php_options: |
+      fastcgi_intercept_errors        on;
+      fastcgi_ignore_client_abort     off;
+      fastcgi_connect_timeout         60;
+      fastcgi_send_timeout            180;
+      fastcgi_read_timeout            180;
+      fastcgi_buffer_size             128k;
+      fastcgi_buffers               4 256k;
+      fastcgi_busy_buffers_size       256k;
+      fastcgi_temp_file_write_size    256k;
+
+                                                                   # ]]]
```

Comment:
With the YAML text block above, the `vim` fold marker will be included in the configuration file. It won't prevent `nginx` from working but looks wonky. Put the `php_upstream` key below the YAML text block to avoid this.